device/pcie/pci_device.cpp

```diff
@@ -255,7 +264,7 @@ PCIDevice::PCIDevice(int pci_device_number, int logical_device_id) :
     logical_id(logical_device_id),
     pci_device_file_desc(open(device_path.c_str(), O_RDWR | O_CLOEXEC)),
     info(read_device_info(pci_device_file_desc)),
-    numa_node(read_sysfs<int>(info, "numa_node")),
+    numa_node(read_sysfs<int>(info, "numa_node", -1)),
```

Comment:
Seeing this line different than others can be confusing. Please leave a comment, you can just copy paste from description.